tasks/02_kubernetes_fundamentals/api_10.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: model-10-deploy
+  labels:
+    app: model-10
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: model-10
+  template:
+    metadata:
+      labels:
+        app: model-10
+    spec:
+      containers:
+      - name: api
+        image: model_api_10
+        imagePullPolicy: Never
+        ports:
+        - containerPort: 5000
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: model-10-service
+spec:
+  selector:
+    app: model-10
+  ports:
+  - port: 6002
+    targetPort: 5000
```

Comment:
1. Please add a newline at the end of the file
2. If you prefer to assign different ports (6001 and 6002) in services leave it like that. However, I want you to know that you don't have to do it ;))